### PR TITLE
Travis-CI problem fix

### PR DESCRIPTION
**Pull request by EspressoCode**
Fixed the Travis problem, I also put back the removed functions into signup.js file and used them for when the user is trying to enter info for creating his/her account.

### CODE REVIEW

File: src/static/js/signup.js [hunk 1]
```diff
@@ -1,59 +1,103 @@
...
         var password = $('#inputPassword').val();
         var confirmPassword = $('#confirmInputPassword').val();
         
-
-        if (userName == null || userName == ''){
-            $('#userNameWarning').text("Please Fill User Name Field");
+        
+        if (ValidateUsername(userName)==false){
+            alert("Please enter a valid Username!");
             error = 1;
```
Comment (Nwitz) [inline src/static/js/signup.js]: We are no longer using alerts, please use the same structure as the  $('#userNameWarning').text("Please Fill User Name Field");

File: src/static/js/signup.js [hunk 2]
```diff
@@ -1,59 +1,103 @@
...
-        
-        if (password != confirmPassword){
-            $('#confirmPasswordWarning').text('Your passwords don\'t match');
+        if (ValidateEmail(email)==false){
+           alert("Please enter a valid email address!");
+            error = 1;
+        }
+        if (ValidatePassword(password)==false){
+            alert("Password should be at least 3 characters including a number and a capital letter!");
+            error = 1;
```
Comment (Nwitz) [inline src/static/js/signup.js]: No alerts please

File: src/static/js/signup.js [hunk 3]
```diff
@@ -1,59 +1,103 @@
...
+            $.post("/signup_submit", JSON.stringify(newUser), function(m_response){
                 alert(m_response.status)
                 switch (m_response.status) {
                     case ServerResponses.SUCCESS:
-                        alert("user successfully created!")
-                        document.location.href = "signin.html"
-                        break;
+                    alert("user successfully created!")
+                    document.location.href = "signin.html"
+                    break;
```
Comment (Nwitz) [inline src/static/js/signup.js]: Indentation inside case should be corrected

**Comment by EspressoCode** [comment]
changes have been made, alerts are now changed to warnings and files are re-indented

**Pull request merged**